Lotto/LottoTest/LottoTest.swift

```diff
@@ -148,7 +186,7 @@ class LottoTest: XCTestCase {
     
     func testSplitLottoNumbers() {
         let input: String = "1, 2, 3, 4, 5, 6"
-        let result: [Int] = StringUtiltity.splitLottoNumbers(to: input)
+        let result: [Int] = try! StringUtiltity.splitLottoNumbers(to: input)
```

Comment:
```suggestion
        let result: Set<Int> = try! StringUtiltity.splitLottoNumbers(to: input)
```
`splitLottoNumbers(to: input)`의 반환 타입이 수정되어 이 부분도 수정이 필요할 것 같습니다.